client/disputes/filters/config.ts

```diff
@@ -47,7 +47,7 @@ export const filters: [ DisputesFilterType, DisputesFilterType ] = [
 			'date_after',
 			'date_between',
 		],
-		showFilters: () => false,
+		showFilters: () => true,
```

Comment:
It was intentionally false. [Here](https://github.com/Automattic/woocommerce-payments/blob/develop/client/deposits/filters/index.js#L28) we enable them based on data fetched from the server side. The intention is to ensure UI elements are not playing hide and show game.

Reply:
Aha! The filter wasn't showing at first, but I believe it's because I didn't have disputes in different currencies. Thank you for the catch! Fixed: https://github.com/Automattic/woocommerce-payments/pull/3731/commits/3594571566a45d8e7b19224845c02b1c66b3235a

Reply:
Yep, thanks!